client/src/hooks/graphql/use-with-upload-status.tsx

```diff
@@ -50,6 +50,8 @@ export function useWithUploadStatus(
   const [uploads, setUploads] = useState<UploadTask[]>([]);
   const [isUploading, setIsUploading] = useState<boolean>(false);
   const [isPolling, setIsPolling] = useState<boolean>(false);
+  const [uploadProcessingText, setUploadProcessingText] =
```

Comment:
for how we're doing this, maybe cleaner if we just provided a poll count here?

```typescript
const [pollStatusCount, setPollStatusCount]
```

...that way all the stuff about ellipses can be a concert for the presentation component, which seems a cleaner division of responsibility

Reply:
3632c1d